### ISSUE TITLE
Cannot unmarshal float environment values.

### ISSUE THREAD

**Issue by cdrage** [kubernetes-incubator/kompose#410]
When passing in a float value through `env:` seems to fail:

Ex:
```
  environment:
    TEST: 4
    BAR: 0.3
    FOO: 0.3
```

```
ERRO Could not parse config for project foobar : Cannot unmarshal '0.3' of type float64 into a string value 
FATA Failed to load compose file: Cannot unmarshal '0.3' of type float64 into a string value 
```

**Comment by surajnarwade** [comment]
@cdrage , I was working on this issue, I found that, this error occurs due to libcompose, so I raised an issue https://github.com/docker/libcompose/issues/435 
(if you passed float value in quotes, it works)

**Comment by surajssd** [comment]
Is this working with docker-compose ?

**Comment by surajnarwade** [comment]
@surajssd , yeah, it works with docker-compose but not with libcompose

**Comment by cdrage** [comment]
@surajnarwade As a work-around, should we update in our code to convert it to a string before being passed in?

**Comment by surajnarwade** [comment]
@cdrage , sounds good actually, I would like to work on it then

**Comment by surajnarwade** [comment]
@cdrage , I tried to work on this, this error occurs due to libcompose, hence as a work around, we can tell user that if you want to put float value, put it in quotes.

**Comment by cdrage** [comment]
@surajnarwade That wouldn't be a great work-around, our code should automatically detect that the environment variables are *not* in quotes and put them in converts / convert to string before passing to libcompose.

I did however take a quick look through the libcompose code and it doesn't seem too daunting to fix it upstream.

**Comment by surajnarwade** [comment]
@cdrage, I tested kompose with your PR on libcompose, https://github.com/docker/libcompose/pull/437
its working.

**Comment by kadel** [comment]
Closing. Fixed in #425
